### DISCUSSION TITLE
Topgrade - How to edit the configuration file?

### DISCUSSION THREAD

**Discussion by gabrielomana** [ScoopInstaller/Scoop#5769] [Help and Questions]
I discovered Scoop while searching for how to use Topgrade on Windows. However, I'm having a hard time knowing how to set it up. On Linux, you edit a TOML file located in ./config/topgrade.toml, but how do you do this on Windows? I read in some places that it should be saved in %APPDATA%, I put it here but when I run it, it doesn't seem to be using this file. My 2 current errors:

- I need to configure topgrade.toml so that it takes the changes and can use winget
![winget](https://github.com/ScoopInstaller/Scoop/assets/36016056/88fb0023-b67b-4270-96e9-ed177d124c3e)

- Error in the Windows update process that I don't know how to solve
![winupdate](https://github.com/ScoopInstaller/Scoop/assets/36016056/502bca13-4cbe-4b61-a92b-c36521e937c0)

Thanks in advance for your support!

**Comment by r15ch13** [comment]
Never used it. Did run it the first time and it has generated the `topgrade.toml` in `%APPDATA%`.
So full path is: `C:\Users\<username>\AppData\Roaming\topgrade.toml`

Got the same windows update error. Added `disable = ["system"]` under `[misc]`

**Reply by niStee** [reply]
A fix has been implemented on https://github.com/topgrade-rs/topgrade/pull/669, https://github.com/topgrade-rs/topgrade/pull/670 and https://github.com/topgrade-rs/topgrade/pull/671. Should be in the next release. Feel free to test it and give me feedback there.
